### ISSUE TITLE
🐞 BUG | Hello Theme – Styling randomly changing un-prompted (text and border color, social media button styling)

### ISSUE THREAD

**Issue by amhjo** [elementor/elementor#12475]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

<p>Hello! 

I have an issue where the styling of certain text and border elements randomly changes to a dark gray (probably the default text color of the Hello Theme), and my social media buttons on the bottom of the page lose all styling altogether (they switch to using the respective button's default colors and shapes). To my knowledge, it's not triggered by anything and randomly happens every few days, and is fixed only by me going into each piece of the template in the Theme Builder to basically do nothing (add and delete a letter etc to be able to click "Update") and click "Update". Voila, the styling is back to normal again for a week or so.

I've tried both un-installing caching plug ins as well as running Autoptimize to add CSS to the <code>head</code>-element, it didn't help. I've also tried regenerating the CSS through Elementor several times with no luck. I'm using the Hello Theme as a base and have now unchecked "disable Elementor's default colors and fonts" as well, hoping it'll help. I posted about this in the Elementor forum and at least two other users had the same issue with the Hello Theme and E3 Pro, so I thought I'd report it since it's quite a devastating bug affecting my reputation as a freelancer since my clients don't understand why the website suddenly looks completely different every few days. I hope you can resolve this bug as quickly as possible!</p>

## Steps to reproduce

<p> I don't know exactly how to reproduce my issue, but I'd wager that using the Hello Theme and creating a basic website using the Theme Builder and changing the colors of the text and certain elements and then leaving it be for a few days and checking back might be enough. Obviously this issue doesn't seem to affect every Hello theme-user, but this is the only reproduction method I can think of. </p>

## Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro). (I can't confirm since it's a client's website, and I don't want to risk losing my design. However, another user in the FB forum is currently troubleshooting to confirm whether it happens with no other plug-ins installedd)
- [ ] This bug happens with a default WordPress theme active. (I can't confirm since it's a client's website, and I don't want to risk losing my design. )
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```

					
== Server Environment ==
	Operating System: FreeBSD
	Software: Apache/2.4.46 (FreeBSD) mpm-itk/2.4.7-04 PHP/7.3.20
	MySQL version: FreeBSD Ports v10.3.23
	PHP Version: 7.3.20
	PHP Max Input Vars: 5000
	PHP Max Post Size: 200M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.5.1
	Site URL: https://www.dintidbeautyclinic.se
	Home URL: https://www.dintidbeautyclinic.se
	WP Multisite: No
	Max Upload Size: 200 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: sv-SE
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.3.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: sv_SE
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36

== Active Plugins ==
	Akismet Anti-Spam
		Version: 4.1.6
		Author: Automattic

	Elementor
		Version: 3.0.7
		Author: Elementor.com

	Elementor Pro
		Version: 3.0.4
		Author: Elementor.com

	Essential Addons for Elementor
		Version: 4.2.3
		Author: WPDeveloper

	Limit Login Attempts Reloaded
		Version: 2.15.2
		Author: WPChef

	Loopia WordPress Plugin
		Version: 0.1
		Author: Loopia AB

	Really Simple SSL
		Version: 3.3.5
		Author: Really Simple Plugins

	Site Kit by Google
		Version: 1.15.0
		Author: Google

	Wordfence Security
		Version: 7.4.11
		Author: Wordfence

	WP Super Cache
		Version: 1.7.1
		Author: Automattic


== Elements Usage ==
	
	footer : 1
		social-icons : 1
		text-editor : 1
	header : 3
		heading : 1
		nav-menu : 2
		social-icons : 1
		theme-site-logo : 2


== Log ==
	
Log: showing 12 of 122020-08-30 15:19:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '2.10.3',
  'to' => '3.0.1',
)]
2020-08-30 17:18:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.3',
  'to' => '3.0.4',
)]
2020-08-31 14:26:03 [info] elementor::elementor_updater Started 
2020-08-31 14:26:03 [info] Elementor/Upgrades - _v_3_0_5_re_move_space_between_widgets_to_kit Start  
2020-08-31 14:26:03 [info] Elementor/Upgrades - _v_3_0_5_re_move_space_between_widgets_to_kit Finished 
2020-08-31 14:26:03 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.4',
  'to' => '3.0.5',
)]
2020-08-31 14:26:03 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.4',
  'to' => '3.0.5',
)]
2020-08-31 14:55:42 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.1',
  'to' => '3.0.2',
)]
2020-09-02 16:10:35 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.2',
  'to' => '3.0.3',
)]
2020-09-07 02:18:50 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.5',
  'to' => '3.0.6',
)]
2020-09-10 00:34:10 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.6',
  'to' => '3.0.7',
)]
2020-09-10 16:22:24 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.3',
  'to' => '3.0.4',
)]

PHP: showing 2 of 2PHP: 2020-08-30 15:24:38 [notice X 21][/www/webvol27/o4/fwgehlbre3qj03e/dintidbeautyclinic.se/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-08-30 19:08:41 [error X 3][/www/webvol27/o4/fwgehlbre3qj03e/dintidbeautyclinic.se/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::281] Uncaught Error: Call to a member function update_meta() on bool in /www/webvol27/o4/fwgehlbre3qj03e/dintidbeautyclinic.se/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:281
Stack trace:
#0 /www/webvol27/o4/fwgehlbre3qj03e/dintidbeautyclinic.se/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(162): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /www/webvol27/o4/fwgehlbre3qj03e/dintidbeautyclinic.se/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(171): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /www/webvol27/o4/fwgehlbre3qj03e/dintidbeautyclinic.se/public_html/wp-includes/class-wp-hook.php(287): Elementor\Core\Common\Modul [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 5 of 5JS: 2020-08-30 16:20:21 [error X 1][https://www.dintidbeautyclinic.se/wp-content/plugins/elementor/assets/js/editor-document.min.js?ver=3.0.3:2:229176] Cannot read property \'map\' of undefined 
JS: 2020-08-31 14:36:05 [error X 34][https://www.dintidbeautyclinic.se/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.5:2:470469] Cannot read property \'document\' of undefined 
JS: 2020-09-01 16:44:02 [error X 1][https://www.dintidbeautyclinic.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5:22:19954] View (cid: \"view61708\") has already been destroyed and cannot be used. 
JS: 2020-09-01 16:44:02 [error X 1][https://www.dintidbeautyclinic.se/wp-content/plugins/elementor/assets/lib/backbone/backbone.marionette.min.js?ver=2.4.5:22:19954] View (cid: \"view61713\") has already been destroyed and cannot be used. 
JS: 2020-09-10 16:28:51 [error X 5][https://www.dintidbeautyclinic.se/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.0.7:2:316365] Cannot read property \'route\' of undefined 


				

```
</details>

**Comment by kbaughan916** [comment]
Hi! I'm having the same issues with the Wordpress Hestia theme. I contacted wordpress support to have them clear my server cache and they found these two errors, stating there's an issue between the theme and the Elementor Plug-in. 

PHP Fatal error:  Uncaught Error: Call to undefined method Elementor\Utils::get_edit_link() in /srv/htdocs/wp-content/themes/hestia/inc/customizer/controls/ui/class-hestia-pagebuilder-button.php:122

PHP Fatal error:  Uncaught Error: Call to undefined method Elementor\Utils::get_edit_link() in /srv/htdocs/wp-content/themes/hestia/inc/customizer/controls/ui/class-hestia-pagebuilder-button.php:122

**Comment by Oshika35** [comment]
Hello,

I'm having the exact same issue, built a page a few days ago, went back to it yesterday and some elements turned to dark grey, they were using global colors. It seems to be random since other elements, also using global colors, work perfectly fine.

**Comment by medienstuermer** [comment]
Same here!

**Comment by heyfletch** [comment]
You are likely suffering from the same issue listed here https://github.com/elementor/elementor/issues/12363

**Comment by arielk** [comment]
This issue has been resolved in Elementor v3.0.8

Feel free to update

Thanks!
